### PR TITLE
Adds Kinect node to master

### PR DESCRIPTION
**Pull request by Kichlids**
Kinect node gets RGB and depth images using Kinect and publishes them

### CODE REVIEW

File: igvc_ws/src/igvc_vision/src/kinect_node.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+def init_kinect_node():
+    global kinect_rgb_pub, kinect_depth_pub
+    rospy.init_node("kinect_node", anonymous = True)
+
+    kinect_rgb_pub = rospy.Publisher('/igvc/kinect_rbg/', Image, queue_size = 10)
+    kinect_depth_pub = rospy.Publisher('/igvc/kinect_depth/', Image, queue_size = 10)
+
+    rospy.Timer(rospy.Duration(secs = 0.005), get_rgb)
+    rospy.Timer(rospy.Duration(secs = 0.005), get_depth)
```
Comment (noahzemlin) [inline igvc_ws/src/igvc_vision/src/kinect_node.py]: This is a 200 Hz polling rate, does the Kinect even update that fast?

**Review by noahzemlin — COMMENTED** [review]
Tested and working, will let @jkleiber review it as well

File: igvc_ws/src/igvc_vision/src/kinect_node.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    
+
+    image_msg = bridge.cv2_to_imgmsg(array)
+    kinect_depth_pub.publish(image_msg)
+    
+
+
+def init_kinect_node():
+    global kinect_rgb_pub, kinect_depth_pub
+    rospy.init_node("kinect_node", anonymous = True)
```
Comment (jkleiber) [inline igvc_ws/src/igvc_vision/src/kinect_node.py]: Why are you making the node anonymous?

File: igvc_ws/src/igvc_vision/src/kinect_node.py (hunk 1 above)
Comment (Kichlids) [reply igvc_ws/src/igvc_vision/src/kinect_node.py]: Changed to 30Hz to match kinect's capabilities.

File: igvc_ws/src/igvc_vision/src/kinect_node.py (hunk 2 above)
Comment (Kichlids) [reply igvc_ws/src/igvc_vision/src/kinect_node.py]: Changed it to anonymous=false

**Pull request closed without merging**